lib/internal/errors.js

```diff
@@ -1429,8 +1431,15 @@ E('ERR_UNKNOWN_MODULE_FORMAT', 'Unknown module format: %s', RangeError);
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s', TypeError);
 E('ERR_UNSUPPORTED_DIR_IMPORT', "Directory import '%s' is not supported " +
 'resolving ES modules imported from %s', Error);
-E('ERR_UNSUPPORTED_ESM_URL_SCHEME', 'Only file and data URLs are supported ' +
-  'by the default ESM loader', Error);
+E('ERR_UNSUPPORTED_ESM_URL_SCHEME', (url) => {
+  let msg = 'Only file and data URLs are supported by the default ESM loader';
+  if (isWindows && url.protocol.length === 2) {
```

Comment:
Why would the protocol length of two be what triggers this? Wouldn't it at the very least be > 1?

Reply:
`.protocol` includes the `:`

Reply:
Yep, so `URL('C:\\example\\foo.mjs')` would have `.protocol === 'c:'`. This covers only such use-case for now.